.github/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM intel/oneapi:latest
+RUN apt update --allow-insecure-repositories
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt install -y g++-13
+RUN apt install -y libgtest-dev
+RUN apt install -y libcxxopts-dev
+
+RUN apt remove --purge --auto-remove -y cmake
+RUN wget https://cmake.org/files/v3.29/cmake-3.29.2.tar.gz
+RUN tar -xzvf cmake-3.29.2.tar.gz && cd cmake-3.29.2 && ./bootstrap && make -j12 && make install
```

Comment:
Do we require to compile CMake from source? This increases the overall CI execution time.

Reply:
Now it is not needed. Ubuntu 24.04 installs sufficiently new cmake. Removed.